sdk-core/src/main/java/dev/restate/sdk/core/InvocationStateMachine.java

```diff
@@ -212,6 +216,16 @@ void close() {
         this.inputSubscription.cancel();
       }
       if (this.outputSubscriber != null) {
+        // Write last message
+        if (suspensionIndexes != null) {
```

Comment:
What if `suspensionIndexes` is empty? The runtime would fail the invocation in this case.

Reply:
the `suspensionIndexes` is never empty. I added an assertion.